.github/version_extractor.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
```

Comment:
Probably overkill for what you are doing here, but sure :)

Reply:
Better safe than sorry.

http://redsymbol.net/articles/unofficial-bash-strict-mode/

Reply:
Nice!